packages/web-components/src/components/va-table/va-table-inner/va-table-inner.tsx

```diff
@@ -306,14 +313,14 @@ export class VaTableInner {
   }
 
   render() {
-    const { tableTitle, tableType, stacked } = this;
+    const { tableTitle, tableType, stacked, isFocusable } = this;
     const classes = classnames({
       'usa-table': true,
       'usa-table--stacked': stacked,
       'usa-table--borderless': tableType === 'borderless',
     });
     return (
-      <div>
+      <div tabIndex={isFocusable ? 0 : undefined}>
```

Comment:
Is adding the `tab-index=0` to this `div` essentially what you're wanting to use to for keyboard tabbing into focus?

I'm going to tag in our a11y specialist Ryan on the PR because I sometimes get confused about when this is needed or not.